spec/data-model.md

```diff
@@ -101,16 +101,16 @@ The `Literal` and `VariableRef` correspond to the the _literal_ and _variable_ s
 When they are used as the `body` of an `Expression`,
 they represent _expression_ values with no _annotation_.
 
-An _unquoted_ value is represented by a `Literal` with `quoted: false`,
-while a _quoted_ value would have `quoted: true`.
+Both _quoted_ and _unquoted_ values are represented by `Literal`,
+as the use or lack of quotation is a presentation detail
+which has no effect on the meaning of the _literal_.
```

Comment:
Perhaps:

```suggestion
`Literal` represents all literal values, both _quoted_ and _unquoted_. 
The presence or absence of quotes is not preserved by the data model
because the quote marks have no meaning.
```

Reply:
With your suggested change, it's unclear to what extent the "have no meaning" phrase applies, which creates some confusion as the marks are a significant presentation detail in the syntax. I do like "presence or absence of quotes" though; that's more appropriately passive than what I had before.

Reply:
Just full stop after data model then?

```suggestion
`Literal` represents all literal values, both _quoted_ and _unquoted_.
The presence or absence of quotes is not preserved by the data model.
```